content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
 
-The Viewport Interaction Model provides the following benefits:
+![Screenshot of 3D Viewport with UI widgets, entity, and a manipulator visible.](/images/user-guide/editor/3d-viewport.png)
 
-+ Improves the ability to select your preferred entity, especially for nested hierarchies or levels that have many entities.
-+ Adds keyboard and mouse actions that you can apply to your selected entities. You can freely switch between local, parent, and world space to change your selected entities.
-+ Establishes an editing pattern that matches that of Component Mode.
+## UI widgets
 
-See the following concepts for the Viewport Interaction Model:
+### Transform mode widget
 
-**Manipulator**
+You can select one of the three transform modes with the transform mode widget, _translate_, _rotate_, or _scale_.  Each transform mode displays a unique [Manipulator](#manipulators) on the entity in the 3D Viewport that you can adjust with the mouse.  Select a transform mode by clicking the widget with the mouse or using a keyboard hotkey listed in the following table. 
 
- With manipulators, you can edit component properties directly in the viewport instead of relying on **Entity Inspector**.
+![Image of transform mode widget](/images/user-guide/editor/transform-widget.png)
 
-**Custom Reference Target**
+| Mode | Keyboard Hotkey | Description |
+| - | - | - |
+| Translate | **1** | Moves an entity along an axis, plane, or surface. |
+| Rotate | **2** | Rotates an entity on one of three axes or in relation to the camera. |
+| Scale | **3** | Uniformly scales an entity's size. |
 
-When you select an entity, you can specify whether to move the entity in its own reference frame, or in relation to another custom reference frame.
+### Reference space widget
 
-## Viewport Interaction Model Limitations 
+Reference spaces set the alignment of manipulators' axes.  By default, entities will translate and rotate in _local space_, which is determined by an entity's **Rotate** and **Translate** values in the [Transform component](\docs\user-guide\components\reference\transform).  _World space_ is constant at all times for all entities.  World space is always centered at the **Translate** coordinates of `(0, 0, 0)` without any rotation.  If an entity is the child of another entity, _parent space_ will reference the **Rotate** and **Translate** values of the parent entity's Transform component.  You can **left-click** on an icon in the widget to lock a reference space selection; **left-click** the icon again to return the reference space to the default, local space.
```

Comment:
nit: update `manipulators'` to just `manipulator`

Reply:
I remember doing a double take at this sentence when I wrote it.  As written, it is the equivalent of "...set the alignment of axes of manipulators".  I'll switch this to a different construction that uses the singular, "...set the alignment of manipulator axes".

Reply:
Yeah that sounds nicer to my ear for some reason, much appreciated